src/common/low_precision_transformations/src/move_fake_quantize.cpp

```diff
@@ -83,7 +84,11 @@ bool MoveFakeQuantize::transform(TransformationContext& context, ov::pass::patte
     if (concat_node == nullptr) {
         return false;
     }
-    const auto concat_axis = concat_node->get_concatenation_axis();
+    int64_t concat_axis = -1;
```

Comment:
why -1 and not the concat_node->get_axis()?

Reply:
Done

Reply:
My understanding is that function `get_concatenation_axis()` returns -1 if rank is dynamic. So current code mimic previous implementation. If we use `get_axis()` directly the behavior of the function will change or it will require additional check to assign -1 in case of dynamic rank. Is it correct or I missed something?

Reply:
With support of analysis by @praasz the suggestion was applied.